hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterRpcClient.java

```diff
@@ -517,7 +517,6 @@ private Object invokeMethod(
           // Communication retries are handled by the retry policy
           if (this.rpcMonitor != null) {
             this.rpcMonitor.proxyOpFailureCommunicate();
-            this.rpcMonitor.proxyOpComplete(false);
```

Comment:
Why we don't need the metric increase anymore?

Reply:
If boolean is false, proxyOpComplete in FederationRPCPerformanceMonitor will do nothing, so maybe it can be deleted?

Reply:
True, I didn't realize that the proxyOpComplete just skips this.
We would need to log the failures here though.
As it is the success flag doesn't make sense.

Reply:
Thanks for review @ayushtkn ,I have added a LOG, could you help review.Thanks!